docs/release-notes/NuGet-6.9.md

```diff
@@ -0,0 +1,121 @@
+---
+title: NuGet 6.9 Release Notes
+description: Release notes for NuGet 6.9 including new features, bug fixes, and DCRs.
+author: jgonz120
+ms.author: jongonza
+ms.date: 2/1/2024
+ms.topic: conceptual
+---
+
+# NuGet 6.9 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.9**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.9](https://visualstudio.microsoft.com/downloads/) | [8.0.200](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.9
+
+* Add `--format`, `--verbosity` , `configfile` options to `dotnet package search` - [#12978](https://github.com/NuGet/Home/issues/12978)
+
+* Add .NET 8 constants to NuGet.Frameworks  - [#12910](https://github.com/NuGet/Home/issues/12910)
+
+* Support for dotnet search command (equivalent to nuget.exe list, later search) - [#6060](https://github.com/NuGet/Home/issues/6060)
+
+* exact match and version search for nuget.exe list command - [#5138](https://github.com/NuGet/Home/issues/5138)
```

Comment:
Both the title of the linked issue, and the title here in the release notes, needs to be changed. What was delivered was not what the title says.

Reply:
I'd just remove it, or maybe combine them in #6060. 

We can ofc combine the 3 issues into 1 description, we don't have to follow a strict rule as long as the issues are linked.